pkg/apis/nsx.vmware.com/v1alpha1/subnetport_types.go

```diff
@@ -19,20 +19,36 @@ type SubnetPortSpec struct {
 type SubnetPortStatus struct {
 	// Conditions describes current state of SubnetPort.
 	Conditions []Condition `json:"conditions,omitempty"`
-	// VIFID describes the attachment VIF ID owned by the SubnetPort in NSX-T.
-	VIFID string `json:"vifID,omitempty"`
-	// IPAddresses describes the IP addresses of the SubnetPort.
-	IPAddresses []SubnetPortIPAddress `json:"ipAddresses,omitempty"`
-	// MACAddress describes the MAC address of the SubnetPort.
-	MACAddress string `json:"macAddress,omitempty"`
+	// Subnet port attachment state.
+	Attachment SegmentPortAttachmentState `json:"attachment,omitempty"`
+	// Subnet gateway.
+	Gateway string `json:"gateway,omitempty"`
 	// LogicalSwitchID defines the logical switch ID in NSX-T.
 	LogicalSwitchID string `json:"logicalSwitchID,omitempty"`
+	// Subnet netmask.
+	Netmask string `json:"netmask,omitempty"`
+	// List of subnet port bindings that are realized.
+	RealizedBindings []AddressBindingEntry `json:"realizedBindings,omitempty"`
 }
 
-type SubnetPortIPAddress struct {
-	Gateway string `json:"gateway,omitempty"`
-	IP      string `json:"ip,omitempty"`
-	Netmask string `json:"netmask,omitempty"`
+// VIF attachment state of a subnet port
+type SegmentPortAttachmentState struct {
+	ID string `json:"id,omitempty"`
+}
+
+type PacketAddressClassifier struct {
```

Comment:
It is called PacketAddressClassifier in NSX API? Do not understand the name..

Reply:
Yes, the description in NSX API:
```
PacketAddressClassifier ([schema](https://x.x.x.x/policy/api_includes/schemas_PacketAddressClassifier.html)) (Deprecated)
Address classifications for a packet

A packet is classified to have an address binding, if its address
configuration matches with all user specified properties.
```
![nsxapi](https://github.com/vmware-tanzu/nsx-operator/assets/13518085/ba954b89-5a0f-4d19-a947-4f6764166eb3)